project.clj

```diff
@@ -129,17 +138,36 @@
                                                      district0x.ui.history.pushroute-hosts "localhost"
                                                      name-bazaar.ui.db.log-level "debug"}
                                    :external-config {:devtools/config {:features-to-install :all}}}}
+
+                       ;; Development on client-side UI, with mainnet
+                       {:id "dev-ui-only"
```

Comment:
I think in `user.clj` you could just get the config map `(figwheel-sidecar.config/fetch-config)` and swap the `name-bazaar.ui.db.environment`in the `:closure-defines` map, without bloating the `project.clj` with another build.

Reply:
Good idea! I'll try this.